app/controllers/SignUpController.scala

```diff
@@ -70,7 +70,13 @@ abstract class SignUpController(val applicationClient: ApplicationClient, cacheC
           )
         },
         data => {
-          val appRoute = ApplicationRoute.withName(data.applicationRoute)
+          val appRouteFromForm = ApplicationRoute.withName(data.applicationRoute)
```

Comment:
That is the value selected by user from the Create application form.
what do you think for `appRouteUserInput` instead? or any other suggestion